### PR TITLE
Update github-workflow.md

### PR DESCRIPTION
**Pull request by franciosi**
Small typo improvements

### CODE REVIEW

File: contributors/guide/github-workflow.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+Description: |
   An overview of the GitHub workflow used by the Kubernetes project. It includes
   some tips and suggestions on things such as keeping your local environment in
   sync with upstream and commit hygiene.
 ---
 
 ![Git workflow](git_workflow.png)
 
-### 1 Fork in the cloud
+### 1 - Fork in the cloud
```
Comment (idvoretskyi) [inline contributors/guide/github-workflow.md:12]: Any reason why these hyphens are required here (and below)?

File: contributors/guide/github-workflow.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
 ---
-title: "GitHub Workflow"
-weight: 6
-description: |
+Title: "GitHub Workflow"
```
Comment (idvoretskyi) [inline contributors/guide/github-workflow.md:2]: No need for this change.

File: contributors/guide/github-workflow.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 
-### 1 Fork in the cloud
+### 1 - Fork in the cloud
 
-1. Visit https://github.com/kubernetes/kubernetes
+1. Visit https://github.com/kubernetes/kubernetes.
 2. Click `Fork` button (top right) to establish a cloud-based fork.
 
-### 2 Clone fork to local storage
+### 2 - Clone fork to local storage
```
Comment (idvoretskyi) [inline contributors/guide/github-workflow.md:17]: See my question above.

File: contributors/guide/github-workflow.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ git remote set-url --push upstream no_push
 git remote -v
 ```
 
-### 3 Branch
+### 3 - Branch
```
Comment (idvoretskyi) [inline contributors/guide/github-workflow.md:65]: See my question above.

File: contributors/guide/github-workflow.md [hunk 5]
```diff
@@ -84,7 +84,7 @@ Then edit code on the `myfeature` branch.
 
 This workflow is process-specific; for quick start build instructions for [kubernetes/kubernetes](https://git.k8s.io/kubernetes) please [see here](/contributors/devel/development.md#building-kubernetes-on-a-local-osshell-environment).
 
-### 4 Keep your branch in sync
+### 4 - Keep your branch in sync
```
Comment (idvoretskyi) [inline contributors/guide/github-workflow.md:87]: See my question above.

**Review by idvoretskyi — CHANGES_REQUESTED** [review]

File: contributors/guide/github-workflow.md (hunk 1 above)
Comment (franciosi) [reply contributors/guide/github-workflow.md:12]: Just to make it easier to understand the steps

File: contributors/guide/github-workflow.md (hunk 2 above)
Comment (franciosi) [reply contributors/guide/github-workflow.md:2]: OK

File: contributors/guide/github-workflow.md (hunk 3 above)
Comment (franciosi) [reply contributors/guide/github-workflow.md:17]: Answered

File: contributors/guide/github-workflow.md (hunk 4 above)
Comment (franciosi) [reply contributors/guide/github-workflow.md:65]: Answered

File: contributors/guide/github-workflow.md (hunk 5 above)
Comment (franciosi) [reply contributors/guide/github-workflow.md:87]: Answered

**Comment by guineveresaenger** [comment]
Hi @franciosi.

I see you have opened two other pull requests to this exact file, #5403 and #5405.

Furthermore, I have seen that you have already [contributed to the Kuberentes ecosystem](https://github.com/kubernetes/test-infra/pull/20445) so please leave typo fixes to other newcomers to learn our pull request workflow.

Additionally, I see from [your open pull requests](https://github.com/issues?q=is%3Apr+is%3Aopen+author%3Afranciosi) that making typo fixes to opensource repositories is the single most common contribution. One typo fix PR to learn about how to contribute to Kubernetes on GitHub is great! More than that, especially broken up into multiple PRs, creates a lot of work for reviewers.

Since I like to assume best intentions, I encourage you to join a SIG, as outlined in this doc, introduce yourself on Kuberentes Slack, and, if writing is your thing, help us on one of our [good first issues](https://github.com/kubernetes/community/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).

/close

**Pull request closed without merging**